pom.xml

```diff
@@ -204,7 +204,12 @@
       <version>${tcnative.version}</version>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
```

Comment:
we can't have this dependency added, otherwise it means we need to file a CQ for 3.4.2